### DISCUSSION TITLE
can risc0 support deep learning model inference ZKP?

### DISCUSSION THREAD

**Discussion by chaoyanghe** [risc0/risc0#287] [Q&A] [answered]

**Comment by brianretford** [comment, marked as answer]
Yes, it can, though we haven't actually done any of it. One could imagine lots of paths to achieving this, including compiling and running ONNX. I think we're a major release or two away from compellingly being able to support AI use cases though - currently computations are limited to a generally insufficient number of cycles (that will be addressed in future  releases though)

**Comment by chaoyanghe** [comment]
@brianretford we are a startup doing web3+AI and very interested in leveraging your library to develop our blockchain system. Would you mind creating a cross-company Slack channel/discord for further discussion?
